authentication-service/src/main/kotlin/com/saveourtool/save/authservice/repository/AuthenticationUserRepository.kt

```diff
@@ -17,24 +16,20 @@ class AuthenticationUserRepository(
      * @param name name of user
      * @return user or null if no results have been found
      */
-    fun findByName(name: String): User? {
-        val record = namedParameterJdbcTemplate.queryForList(
-            "SELECT * FROM save_cloud.user WHERE name = :name",
-            mapOf("name" to name)
-        ).singleOrNull()
-            .orNotFound {
```

Comment:
I think we need some logging here

Reply:
Logging, it's ok. But now we throw exception instead of Spring Security.
Will add logging when `Mono` is empty without exception